core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1620,8 +1766,9 @@ class SparkContext(config: SparkConf) extends Logging {
 
   /**
    * :: DeveloperApi ::
-   * Return information about what RDDs are cached, if they are in mem or on disk, how much space
-   * they take, etc.
```

Comment:
If you want to remove `Return` or make the description into `@returns`, I guess it should be at least consistent. It seem https://github.com/apache/spark/pull/16137/files#r91009309 is a bit different.